code/parse/parselo.cpp

```diff
@@ -3807,6 +3812,108 @@ int split_str(const char *src, int max_pixel_w, SCP_vector<int> &n_chars, SCP_ve
 	return line_num;
 }
 
+// A narrower but much faster alternative to split_str(), takes a string and a max pixel length, returns a vector with
+// one string per line. Does not currently support a max line count or ignoring of characters.
+SCP_vector<SCP_string>
+str_wrap_to_width(const SCP_string& source_string, int max_pixel_length, bool strip_leading_whitespace)
+{
+	// To avoid any unexpected side effects, we're copying the orignal string.
+	SCP_string new_string = SCP_string(source_string);
+
+	SCP_vector<SCP_string> lines = SCP_vector<SCP_string>();
+
+	while (strip_leading_whitespace && new_string.length() > 0 && is_white_space(new_string[0])) {
+		new_string.erase(0, 1);
+	}
+	if (new_string.length() == 0)
```

Comment:
This can check for `empty()` rather than `length() == 0`.  Ditto elsewhere in this function